### PR TITLE
jetty: add license

### PR DESCRIPTION
**Pull request by janesmae**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by Rylan12 — CHANGES_REQUESTED** [review]
Thanks for the PR, @janesmae!

File: Formula/jetty.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class Jetty < Formula
   url "https://search.maven.org/remotecontent?filepath=org/eclipse/jetty/jetty-distribution/9.4.32.v20200930/jetty-distribution-9.4.32.v20200930.tar.gz"
   version "9.4.32.v20200930"
   sha256 "64e5fe203347a4b4f5b5a828f9c241b2f13f154b56babccd533513991fd156c2"
+  license "Apache-2.0"
```
Comment (Rylan12) [inline Formula/jetty.rb:7]:
```suggestion
  license any_of: ["Apache-2.0", "EPL-1.0"]
```

From the readme:

> Jetty is open source and is dual licensed using the Apache 2.0 and
> Eclipse Public License 1.0.   You may choose either license when
> distributing Jetty.

**Review by Rylan12 — CHANGES_REQUESTED** [review]

File: Formula/jetty.rb (hunk 1 above)
Comment (janesmae) [reply Formula/jetty.rb:7]: ty, just figured out the dual license myself as well

**Comment by Rylan12** [comment]
Thanks, @janesmae!

One more thing: can you please squash your changes into one commit with the same name as the PR title (`jetty: add license`)?

**Comment by janesmae** [comment]
> Thanks, @janesmae!
> 
> One more thing: can you please squash your changes into one commit with the same name as the PR title (`jetty: add license`)?

I'm sorry if this is a beginner question but how? Make a new PR with just a single commit?

**Comment by Rylan12** [comment]
> I'm sorry if this is a beginner question but how? Make a new PR with just a single commit?

No worries! You'll need to do an interactive rebase to squash the commits into one and then force push back to this branch. Take a look at [the docs](https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request#following-up) for a basic guide.

Here's a [quick guide](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec) to interactive rebases which might be helpful.

If you can't figure it out, just give me a ping and I'll handle it 😄

**Comment by janesmae** [comment]
> > I'm sorry if this is a beginner question but how? Make a new PR with just a single commit?
> 
> No worries! You'll need to do an interactive rebase and squash the commits into one. You'll then need to force push back to this branch. Take a look at [the docs](https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request#following-up) for a basic guide.
> 
> Here's a [quick guide](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec) to interactive rebases which might be helpful.
> 
> If you can't figure it out, just give me a ping and I'll handle it 😄

thank you for the help

**Review by Rylan12 — APPROVED** [review]

**Comment by Rylan12** [comment]
Thanks, @janesmae!

Just FYI, I renamed the commit to `jetty: add license` to fit with our commit name preferences.

**Pull request merged**